src/redux/slices/walletconnect.js

```diff
@@ -60,13 +57,16 @@ export const walletConnectRemovePendingRedirect = createAsyncThunk(
 
 export const walletConnectOnSessionRequest = createAsyncThunk(
   'walletconnect/onSessionRequest',
-  async ({ uri }, { dispatch, getState }) => {
+  async ({ uri, requestId }, { dispatch, getState }) => {
     try {
       const { clientMeta } = await getNativeOptions();
+      const isPrelocked = () => getState().keyring.isPrelocked;
+      const isUnlocked = () => getState().keyring.isUnlocked;
+      const isInitialized = () => getState().keyring.isInitialized;
```

Comment:
check if you can convert this into constants instead of functions

Reply:
We need functions to have the values updated 😞